### PR TITLE
Considerations update

### LINKED ISSUE #378: Add PRA guidance to the 18F UX Guide
### A Description of work
Tactical information about how to get PRA approval and the fast track process is not mentioned in the ux guide. PRA is mentioned several times in the guide but it could be more helpful.

**references for review**
- https://pra.digital.gov/
- slide 35-36 of this slide [deck](https://docs.google.com/presentation/d/15UVLBnmlsKnFBQmjAVWRe9HntSDCsq12ZTjlKovVS98/edit#slide=id.g20c56d5d7ab_5_38)
- 2022 PRA review [doc](https://docs.google.com/document/d/1r0-AQ6DChw3FoZEV9q6K9dnhEa05uwLyPcEzLAV74FI/edit)
- notes and recording from this [research guild session](https://docs.google.com/document/d/1QFreBLN_AwDJi5ooIwO3xFGADMat5G8hyAw1drCjyTw/edit)

### Point of contact on this issue
You can reach out to @bpdesigns 

**Billable?**

- [ ] Yes
- [x] No
If yes, tock code:

**Skills needed**

- [x] Any human
- [ ] Design
- [ ] Content
- [ ] Engineering
- [ ] Acquisition
- [ ] Product
- [ ] Other

**Timeline**
Does this need to happen in the next two weeks?

- [ ] Yes
- [x] No

How much time do you anticipate this work taking?
-  under two weeks

### Tasks 
- [x] review references (above)
- [x] create a doc with updated content
- [x] share doc with @bpdesigns and the #g-research slack channel for review
- [ ] implement approved changes. 

### Acceptance criteria
- [ ] Updated PRA guidance is added to the Plan and Legal pages of the UX Guide.

### PR DESCRIPTION
**Pull request by egamble23**
## Description of changes

> Adding subsection on PRA tactical considerations, regarding #378 . 

## Related issues

> Closes #378 

## Reviewers:

> Ben Peterson for final review of content before merging. 

- [ ] @bpdesigns

### CODE REVIEW

File: _pages/research/legal.md [hunk 1]
```diff
@@ -50,6 +52,86 @@ Some tweet-length examples:
...
+
+Note, **voluntary collections are not automatically exempt from PRA clearance**. [Contact your agency’s PRA contact](https://pra.digital.gov/contact/) to answer more in-depth questions.
+
+### Full PRA
+
+The full PRA process is outlined on the [PRA website](https://pra.digital.gov/clearance-process/). 
+
+Using the PRA process, a new information collection request (ICR) can take an estimated 9 to 12 months to complete. 
+
+### PRA-exempt and -fast track
```
Comment (bpdesigns) [inline _pages/research/legal.md:96]: Should this be "fast-track"?

**Review by bpdesigns — CHANGES_REQUESTED** [review]
Hi Erin! This looks great. Just left one note about a heading.

File: _pages/research/legal.md (hunk 1 above)
Comment (egamble23) [reply _pages/research/legal.md]: Thanks for the note! I was using "fast track" as a noun. I've adjusted it to be a modifier. Please have a look when you get a chance.

**Comment by egamble23** [comment]
Thanks, @bpdesigns ! I made a text edit based on your note. Please have a look when you have a moment.

**Review by bpdesigns — APPROVED** [review]

**Comment by egamble23** [comment]
Thanks so much, @bpdesigns ! I will notify stakeholders of the page's updates and to close the loop. Thanks so much for your guidance and review!

**Pull request merged**